pom.xml

```diff
@@ -248,7 +248,7 @@
     <repository>
       <id>mapr-repo</id>
       <name>MapR Repository</name>
-      <url>http://repository.mapr.com/maven</url>
+      <url>http://repository.mapr.com/maven/</url>
```

Comment:
I think this was an unrelated change. You also need to give this PR a proper title. Did you review https://cwiki.apache.org/confluence/display/SPARK/Contributing+to+Spark first?